musicbrainz-postgresql/recipes/pgbouncer.rb

```diff
@@ -2,8 +2,8 @@
 
 package "pgbouncer"
 
-template "/etc/pgbouncer/userlist.txt"
-template "/etc/pgbouncer/pgbouncer.ini"
+template "/usr/local/pgbouncer/etc/auth_file"
+template "/usr/local/pgbouncer/etc/pgbouncer.ini"
 
 execute "Enable pgbouncer" do
   command "/bin/sed s/START=0/START=1/ -i /etc/default/pgbouncer"
```

Comment:
This whole section seems to be unnecessary, if we're doing a daemontools thing (as we appear to be). But I don't see anything in this file that's making it work that way nor installing it to the /usr/local folder where it's being used, since the package installs it to /usr/sbin: http://packages.ubuntu.com/precise/amd64/pgbouncer/filelist

So the former state of this appears to be consistent with itself, where the version after the change is not, but obviously the former setup wasn't controlling this. So this seems like an incorrect change overall to me, but I'm not sure what the correct way to go about this is -- presumably making it properly set up the /usr/local version and the daemontools service, or making our actual setup use the packaged version.